voluncheer/map/models.py

```diff
@@ -1,3 +1,18 @@
-# from django.db import models
+from django.db import models
+
 
 # Create your models here.
+class NYCharities(models.Model):
+    class Meta:
+        verbose_name_plural = "NYCharities"
+
+    name = models.CharField(max_length=255)
+    street = models.CharField(max_length=255)
+    city = models.CharField(max_length=255)
+    state = models.CharField(max_length=2)
+    type = models.CharField(max_length=255)
+    latitude = models.FloatField(default=0.0)
```

Comment:
WDYT about using the models.PointField for the latitude and longitude? The GeoDjango model API might be useful here.

https://docs.djangoproject.com/en/4.1/ref/contrib/gis/model-api/

Reply:
When working on the opportunity model, my initial research gave me the impression that GeoDjango was more than is necessary for our app, but I can't really point at any reason why. 

If we are only using the the longitude and latitude to plot a point on a map, it may be simpler to just use separate float or decimal fields for lon and lat as you have already. This would at the very least save us from importing GeoDjango models in addition to vanilla django.db models.

Reply:
Btw I landed on using decimal fields for lon and lat based on this: https://stackoverflow.com/questions/30706799/which-model-field-to-use-in-django-to-store-longitude-and-latitude-values

The main takeaway seems to be that float, decimal and point are all viable options.

Reply:
Ack, thanks for the explanation Doug. I just wanted to be sure we were considering all the options. Decimal seems like the best option. Float is always a bit scary to me given its inherent inaccuracy.

Reply:
I saw that too @dbleek! Thanks for sending the link. I didn't think the GeoDjango was necessary since we are using Google Maps